include/constants/songs.h

```diff
@@ -1,64 +1,103 @@
 #pragma once
 
 enum song_idx {
-    SONG_01 = 0x01,
-    SONG_02 = 0x02,
-    SONG_03 = 0x03,
-    SONG_BGM_WMAP_01 = 0x004, // "The Beginning"
-    SONG_BGM_WMAP_02 = 0x005, // "Treasured Memories"
-    SONG_BGM_WMAP_03 = 0x006, // "Grim Journey"
-    SONG_BGM_WMAP_04 = 0x007, // "Ties of Friendship"
-    SONG_BGM_WMAP_05 = 0x008, // "Ray of Hope"
-    SONG_09 = 0x09,
-    SONG_0A = 0x0A,
-    SONG_0C = 0x0C,
-    SONG_0E = 0x0E,
-    SONG_0F = 0x0F,
-    SONG_BGM_MAP_PL10 = 0x010,
-    SONG_11 = 0x11,
-    SONG_12 = 0x12,
-    SONG_13 = 0x13,
-    SONG_14 = 0x14,
-    SONG_22 = 0x22,
-    SONG_23 = 0x23,
-    SONG_24 = 0x24,
-    SONG_25 = 0x25,
-    SONG_26 = 0x26,
-    SONG_27 = 0x27,
-    SONG_28 = 0x28,
-    SONG_29 = 0x29,
-    SONG_2A = 0x2A,
-    SONG_2B = 0x2B,
-    SONG_2C = 0x2C,
-    SONG_BGM_EVT_05 = 0x02F, // "Lights in the Dark"
-    SONG_31 = 0x31,
-    SONG_32 = 0x32,
-    SONG_33 = 0x33,
-    SONG_BGM_SELECT = 0x034, // "Combat Preparation"
-    SONG_37 = 0x37,
-    SONG_3B = 0x3B,
-    SONG_3C = 0x3C,
-    SONG_3D = 0x3D,
-    SONG_3E = 0x3E,
-    SONG_3F = 0x3F,
-    SONG_43 = 0x43,
-    SONG_44 = 0x44,
-    SONG_45 = 0x45,
-    SONG_BGM_ED_AFTER = 0x046,
-    SONG_BGM_ED_STAFF_2 = 0x056,
+    SONG_NONE = 0x00,
+    SONG_THE_VALIANT = 0x01,
+    SONG_MAIN_THEME = 0x02,
```

Comment:
`0x02` could be `SONG_MAIN_THEME_EXT`, since it is the extended version that plays after the intro cutscene finishes. You can refer to `titlescreen.c` in function `Title_Loop_DrawRedBlueOrbs` (`FE8U:0x080C5C64`)